src/wrappers/themis/rust/src/keys.rs

```diff
@@ -0,0 +1,516 @@
+// Copyright 2018 (c) rust-themis developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Cryptographic keys.
+//!
+//! This module contains data structures for keys supported by Themis: RSA and ECDSA key pairs.
+
+use std::fmt;
+
+use bindings::{themis_get_key_kind, themis_is_valid_key};
+use error::{Error, ErrorKind, Result};
+use utils::into_raw_parts;
+
+/// Key material.
+#[derive(Clone, Eq, PartialEq, Hash)]
+pub(crate) struct KeyBytes(Vec<u8>);
+
+// TODO: securely zero memory when dropping KeyBytes (?)
```

Comment:
will be good if you add it now. it's really important and easy in rust as I understand. to be sure that keys will be filled with zeroes

Reply:
Thanks for good catch!
Zeroing private keys is gonna be part of next PR